### ISSUE TITLE
add UI font size guidelines for developers

### ISSUE THREAD

**Issue by jryio** [gregsadetsky/rctv#33]
The primary target for an RCTV app is the physical display in the hub. Given that, the minimum font size should be: `X` and the recommended font size should be `Y`.

If you are adding a third party app (a URL which you did not author) please use the `scale` field when adding your app to https://rctv.recurse.com/develoeprs

**Comment by gregsadetsky** [comment]
yeah for sure it's sort of a problem that it's hard to simulate apps and to be thinking about 1) the 90 degree rotate screen i.e. huge height restricted width 2) the fact that it's both a giant tv but also people are looking at it not necessarily from 30 inches away

yeah, very good point. maybe making a side page about app guidelines - with the sdk docs and design recommendations -- would be good!
